test/fgpu/test_engine.py

```diff
@@ -766,8 +766,9 @@ async def test_delay_changes(
     @pytest.mark.parametrize("channels", [64, 2048, 8192])
     # Use small spectra-per-heap to get finer-grained testing of which spectra
     # were ditched. Fewer would be better, but there are internal alignment
-    # requirements.
-    @pytest.mark.cmdline_args("--spectra-per-heap=32")
+    # requirements. --src-chunk-samples needs to be increased to ensure
+    # narrowband windows fit.
+    @pytest.mark.cmdline_args("--spectra-per-heap=32", "--src-chunk-samples=4194304")
```

Comment:
Again, I appreciate the comment, but how did you arrive at this number for `src-chunk-samples`?
* It's eight times larger than `CHUNK_SAMPLES` declared here/at the top, but still a quarter of the default in `main.py`.

EDIT: I guess the comment "needs to be increased" might need to updated to "needs to be increased *over the default used here". I read it as an increase relative to the `main.py` default.

Reply:
Ok, will update the comment.